lms/envs/common.py

```diff
@@ -311,7 +311,7 @@
     'ENABLE_COURSE_SORTING_BY_START_DATE': False,
 
     # Flag to enable new user account APIs.
-    'ENABLE_USER_REST_API': False,
+    'ENABLE_USER_REST_API': True,
```

Comment:
We have to set this back to False if we merge this branch to master ahead of UI being done!